migrations/sql-migrations/knexfile.js

```diff
@@ -19,16 +19,18 @@ const fetchConfiguration = async (environment, sandboxId, region) => {
   };
 };
 
-module.exports = async (env, inputSandboxId) => {
+module.exports = async (env, inputSandboxId, inputRegion, inputLocalPort, profile) => {
```

Comment:
Have you tested that this works well when you do `make migrate-down && make migrate` in localhost? 

Also that it works well in a staging (nondefault) environment? (you can start up your own sandbox id rds cluster by using the `biomage stage` command with `--with_rds` flag)

Reply:
I checked `make migrate-down && make migrate` locally -- works fine.

Not sure what it is I need to check in a staging environment. That I can migrate an experiment to/from?

Reply:
> works for an experiment in my account from Biomage production to HMS production.

ah ok, then it's probably fine :P